lib/shoot/ngrok.rb

```diff
@@ -7,8 +7,12 @@ class Ngrok
     extend Forwardable
     def_delegators :@process, :start, :stop, :exited?
 
-    def initialize(port = 3000)
-      @process = ChildProcess.build("ngrok", "-log=stdout", "-subdomain=#{subdomain}", port.to_s)
+    def initialize(options)
+      port = options[:port] || 3000
+      params = ["ngrok", "http", port.to_s, "-log=stdout", "-subdomain=#{subdomain}"]
```

Comment:
since the syntax for ngrok changed.
1.x: "ngrok 3000 <params>"
2.x: "ngrok http 3000 <params>"